pyproject.toml

```diff
@@ -348,6 +350,7 @@ skips = [
     "B101", # assert_used
     "B105", # hardcoded_password_string too noisy on defaults values empty string
     "B106", # hardcoded_password_funcarg too noisy on defaults values empty string
+    "B404", # import_subprocess
```

Comment:
Instead of blank skip, please do where it's needed using `# nosec: import_subprocess`